src/lib/tvgRadialGradient.cpp

```diff
@@ -64,8 +64,6 @@ RadialGradient::~RadialGradient()
 
 Result RadialGradient::radial(float cx, float cy, float radius) noexcept
 {
-    if (radius < FLT_EPSILON) return Result::InvalidArguments;
```

Comment:
we don't allow minus value here.
so you can change like this:
if (radius < 0) return Result::InvalidArguments;